cumulus/polkadot-parachain/src/command.rs

```diff
@@ -120,15 +133,17 @@ fn runtime(id: &str) -> Runtime {
 		Runtime::Coretime(
 			id.parse::<chain_spec::coretime::CoretimeRuntimeType>().expect("Invalid value"),
 		)
-	} else if id.starts_with("glutton-westend") {
-		Runtime::GluttonWestend
 	} else if id.starts_with("glutton") {
 		Runtime::Glutton
 	} else if id.starts_with(chain_spec::people::PeopleRuntimeType::ID_PREFIX) {
 		Runtime::People(id.parse::<chain_spec::people::PeopleRuntimeType>().expect("Invalid value"))
 	} else {
-		log::warn!("No specific runtime was recognized for ChainSpec's id: '{}', so Runtime::default() will be used", id);
-		Runtime::default()
+		log::warn!(
```

Comment:
It seems minor, but this is a breaking change. There seems to be opposing views to "fork off" omninode and just deprecate polkadot-parachain once omninode is ready. In my point of view this would actually still be the best approach, because then we can merge/break as much as we want until it is ready and stable, where we need to become more careful again.

If we really want to stick with overriding polkadot-parachain, I would either:

1. Make sure the user interface stays stable until everything is ready. (All omninode features will be accessible via things like --experimental-omninode-foo)
2. Keep changes in a branch until we are fully ready.

I don't think that it would be a good idea to break and change the interface all the time along the way on an already existing product. This is precisely what we are hated for.

I mean in this particular case we issued a warning, so I guess users were not expected to have used this default mode, but the general remark remains ... also likely people did it anyways and ignored the warning.

Reply:
> It seems minor, but this is a breaking change

What breaking change?

Reply:
Uh yeah. Ok, while the code is duplicated in the end it is executing exactly the same. Then please disregard.